app/controllers/exchanges/hbx_profiles_controller.rb

```diff
@@ -914,9 +914,10 @@ def uniq_cancel_params
   def get_resource(params)
     return nil if params[:resource_id].blank?
 
-    if params[:resource_name].classify.constantize == Person
+    case params[:resource_name]
+    when 'Person'
       Person.find(params[:resource_id])
-    else
+    when 'BenefitSponsors::Organizations::AcaShopDcEmployerProfile'
```

Comment:
I think we should leave the 'else' case as it is.  I'm not sure that this is only used for 'BenefitSponsors::Organizations::AcaShopDcEmployerProfile' - I think it might be used for anything that can be a subclass of profile.  Like couldn't this also be used to message like broker profiles or other kinds of profiles?